webhook/hook.go

```diff
@@ -42,15 +45,20 @@ func asJSON(w http.ResponseWriter, status int, message string) {
 }
 
 func checkBearerToken(r *http.Request, c config.Configuration) error {
-	tokenHeader := r.Header.Get("Authorization")
-	if tokenHeader == "" {
-		return fmt.Errorf("Request does not have Authorization header")
-	}
-	headerElems := strings.Split(tokenHeader, " ")
-	if len(headerElems) != 2 || (len(headerElems) > 0 && headerElems[0] != "Bearer") {
-		return fmt.Errorf("Malformed Authorization header")
+	tokenHeader := r.Header.Get(AuthorizationHeader)
+	tokenQuery := r.URL.Query().Get("token")
+	var token string
+	if tokenHeader != "" {
+		headerElems := strings.Split(tokenHeader, " ")
+		if len(headerElems) != 2 || (len(headerElems) > 0 && headerElems[0] != "Bearer") {
+			return fmt.Errorf("Malformed Authorization header")
+		}
+		token = headerElems[1]
+	} else if tokenQuery != "" {
+		token = tokenQuery
+	} else {
+		return fmt.Errorf("Request does not have bearer token")
```

Comment:
The _bearer_ might be associated with the Authorization header too much. I would phrase this as:

Request dos not contain an Authorization token

In general I have the impression that we can improve the wording around _Bearer_, _Authorization_, _token_ and the like. Reading through the code is somewhat confusing when it comes to those terms and the context they are used in.